guide/popular-topics/permissions.md

```diff
@@ -1,8 +1,8 @@
 # Permissions
 
-Permissions are Discords primary feature enabling users to customize the workings of their server to their liking.
+Permissions are Discords primary feature, enabling users to customize the workings of their server to their liking.
 To break it down to essentials: Permissions and permission overwrites tell Discord who is allowed to do what and where.
-When first confronted with them they can be quite confusing, but no worries we are here to take care of that, so let's dive in!
+When first confronted with them, they can be quite confusing, but no worries, we are here to take care of that, so let's dive in!
```

Comment:
This needs a whole rewrite, not just adding commas. `Permissions can be very confusing at first, but this guide is here to explain and clarify them, so let's dive in!` or something like that.

Reply:
I'll change it to this for now, but feel free to suggest changes again if you're (or someone else) is unhappy with it.